test/UnitTimeoutInactive.cpp

```diff
@@ -37,7 +37,7 @@ class UnitTimeoutInactivity : public UnitTimeoutBase0
     void configure(Poco::Util::LayeredConfiguration& /* config */) override
     {
         // net::Defaults.inactivityTimeout = std::chrono::seconds(3600);
-        net::Defaults.inactivityTimeout = std::chrono::milliseconds(100);
+        net::Defaults.inactivityTimeout = std::chrono::milliseconds(150);
```

Comment:
Aha, the default is 3600 seconds and we decrease this drastically to have a faster test, but then with 100ms we turn a stream socket into an inactive one so fast that it happens when no timeout should happen in a debug build, and increasing that decreased value makes the test more stable, while still not having to wait for 1h for the inactivity to happen. Makes sense after reading this about 3 times. :-)